### PR TITLE
Feature #813 - Dashboard DOM tests + Dummy data update

### LINKED ISSUE #813: Dashboard route: DOM Tests

### PR DESCRIPTION
**Pull request by abefernan**
Closes #813.

Does not cover issues documented in #833.

### CODE REVIEW

File: packages/sil-governance/src/components/BlockchainTime/index.tsx [hunk 1]
```diff
@@ -3,6 +3,8 @@ import { Block, Typography } from "medulas-react-components";
 import React from "react";
 import { ReadonlyDate } from "readonly-date";
 
+export const blockchainTimeHtmlId = "blockchain-time";
```
Comment (albertandrejev) [inline packages/sil-governance/src/components/BlockchainTime/index.tsx]: Please use capital letters for this kind of constants. Like BLOCKCHAIN_TIME_HTML_ID

File: packages/sil-governance/src/routes/dashboard/components/Proposal/CollapsingData/ReadMore.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+interface Props {
+  readonly description: string;
+  readonly toggleExpanded: () => void;
+}
+
+const ReadMore = ({ description, toggleExpanded }: Props): JSX.Element => {
+  return (
+    <Block marginTop={2}>
+      <Description fullDescription={description} expanded={false} />
```
Comment (albertandrejev) [inline packages/sil-governance/src/routes/dashboard/components/Proposal/CollapsingData/ReadMore.tsx]: What do you think if you will pass description as `children` element?

File: packages/sil-governance/src/routes/dashboard/components/Proposal/CollapsingData/ReadMore.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import { Block, Typography } from "medulas-react-components";
+import React from "react";
+
+import Description from "./Description";
+
+interface Props {
+  readonly description: string;
```
Comment (albertandrejev) [inline packages/sil-governance/src/routes/dashboard/components/Proposal/CollapsingData/ReadMore.tsx]: Also you can pass description as `children` too.

File: packages/sil-governance/src/routes/dashboard/index.dom.spec.ts [hunk 4]
```diff
@@ -66,36 +98,157 @@ describe("DOM > Feature > Dashboard", () => {
...
+  it("has a 'Blockchain time' display", async () => {
+    const blockchainTimeLabel = await getBlockchainTimeLabel(dashboardDom);
+    expect(blockchainTimeLabel).toBe("Blockchain time");
   }, 60000);
 
-  it("has an ended proposal with correct voting panel", async () => {
-    const endedProposal = (await getProposals(dashboardDom))[0];
-    checkEndedVotingPanel(endedProposal);
+  // NOTE not able to change SelectField value with input()
+  it.skip("has a sorting menu that sorts by expiry date, start date, or vote", async () => {
```
Comment (albertandrejev) [inline packages/sil-governance/src/routes/dashboard/index.dom.spec.ts]: Is this `skip` expected?

File: packages/sil-governance/src/components/BlockchainTime/index.tsx (hunk 1 above)
Comment (abefernan) [reply packages/sil-governance/src/components/BlockchainTime/index.tsx]: @webmaster128 told me to try to move away from that convention.

File: packages/sil-governance/src/routes/dashboard/index.dom.spec.ts (hunk 4 above)
Comment (abefernan) [reply packages/sil-governance/src/routes/dashboard/index.dom.spec.ts]: No. I'm not able to test input select fields with `input()`, which uses `TestUtils.simulate`, so the test doesn't work. Maybe I should try using `click()` and clicking on the dropdown option I want.

**Pull request merged**